spec/registry.xml

```diff
@@ -15,21 +15,25 @@
 
     <formatSignature>
       <input validationRule="iso8601"/>
-      <!-- The predefined date formatting style to use. -->
-      <option name="dateStyle" values="full long medium short"/>
-      <!-- The predefined time formatting style to use. -->
-      <option name="timeStyle" values="full long medium short"/>
-      <!-- Calendar to use. -->
-      <option name="calendar" values="buddhist chinese coptic dangi ethioaa ethiopic gregory hebrew indian islamic islamic-umalqura islamic-tbla islamic-civil islamic-rgsa iso8601 japanese persian roc"/>
-      <!-- Numbering system to use. -->
-      <option name="numberingSystem" values="arab arabext bali beng deva fullwide gujr guru hanidec khmr knda laoo latn limb mlym mong mymr orya tamldec telu thai tibt"/>
-      <!-- The time zone to use. The only value implementations must recognize
-         is "UTC"; the default is the runtime's default time zone.
-         Implementations may also recognize the time zone names of the IANA
-         time zone database, such as "Asia/Shanghai", "Asia/Kolkata",
-         "America/New_York".
-      -->
-      <option name="timeZone" validationRule="timeZoneId"/>
+      <option name="dateStyle" values="full long medium short">
+        The predefined date formatting style to use.
+      </option>
+      <option name="timeStyle" values="full long medium short">
+        The predefined time formatting style to use.
+      </option>
+      <option name="calendar" values="buddhist chinese coptic dangi ethioaa ethiopic gregory hebrew indian islamic islamic-umalqura islamic-tbla islamic-civil islamic-rgsa iso8601 japanese persian roc">
```

Comment:
Which source did you use for these? We should have a pointer as this list is not likely to be stabilized.

Reply:
I only moved the description into the body here; the values were added by @mihnita in #420; I presume that their source is the ECMA-402 spec, which may itself refer to another source.